core/src/main/resources/error/error-classes.json

```diff
@@ -929,6 +929,11 @@
       "Star (*) is not allowed in a select list when GROUP BY an ordinal position is used."
     ]
   },
+  "STREAM_FAILED" : {
+    "message" : [
+      "Query [id = <id>, runId = <runId>] terminated with exception: <message>"
```

Comment:
I leaved the error message as is to don't break existing apps but the `message` can be removed since it is available in the `cause` exception.